Z80_CPU.cpp

```diff
@@ -129,11 +129,11 @@ void Z80_CPU::LD_FROM_REGISTER_TO_REGISTER(int8_t &reg1, int8_t &reg2)
 	Load to 8-bit register B from immediate data n
 */
 
-void Z80_CPU::LD_B_n()
+void Z80_CPU::LD_IMMEDIATE_DATA_TO_REGISTER(int8_t &reg)
```

Comment:
Can you specify what size data you will be fetching in the name ?
something like `LD_IMMEDIATE_8BIT_DATA_TO_REGISTER`, since we're going to have another function that fetch 16-bit data.